doc/manual/decoding.rst

```diff
@@ -27,6 +27,11 @@ Scikit-learn API enforces the requirement that data arrays must be 2D. A common
 
 To recover the original 3D data, an ``inverse_transform`` can be used. The ``epochs_vectorizer`` is particularly useful when constructing a pipeline object (used mainly for parameter search and cross validation). The ``epochs_vectorizer`` is the first estimator in the pipeline enabling estimators downstream to be more advanced estimators implemented in Scikit-learn. 
 
+Vectorizer
+^^^^^^^^^^
+MNE offers broad range of features allowing processing of M/EEG data. These processing steps could be chained or pipelined(see scikit-learn pipeline) allowing for parameter search and cross validation, making life easier. However since different(processed) data have dofferent dimensions, to make it consistent ``Vectorizer`` can be used. This could be placed in the beginning step of pipeline.
```

Comment:
Evoked is useful across subjects, we may want to support lists of evoked.

On Wed, May 11, 2016 at 7:54 PM Jean-Rémi KING <notifications@github.com>
wrote:

> In doc/manual/decoding.rst
> <https://github.com/mne-tools/mne-python/pull/3212#discussion_r62894725>:
>
> > @@ -27,6 +27,11 @@ Scikit-learn API enforces the requirement that data arrays must be 2D. A common
> >
> >  To recover the original 3D data, an ``inverse_transform`` can be used. The ``epochs_vectorizer`` is particularly useful when constructing a pipeline object (used mainly for parameter search and cross validation). The ``epochs_vectorizer`` is the first estimator in the pipeline enabling estimators downstream to be more advanced estimators implemented in Scikit-learn.
> >
> > +Vectorizer
> > +^^^^^^^^^^
> > +MNE offers broad range of features allowing processing of M/EEG data. These processing steps could be chained or pipelined(see scikit-learn pipeline) allowing for parameter search and cross validation, making life easier. However since different(processed) data have dofferent dimensions, to make it consistent ``Vectorizer`` can be used. This could be placed in the beginning step of pipeline.
>
> pipelined (
> cross-validation
> remove "making life easier" .
> dofferent => different.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3212/files/4e77003ad08d76febacba216027ab0b99d3fa4fa#r62894725>
>